docs/section11/Rule11-13.md

```diff
@@ -0,0 +1,34 @@
+# Service_Water_Heating - Rule 11-13  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** FALSE  
+
+**Rule ID:** 11-13  
+
+**Rule Description:** "Service water-heating energy consumption shall be calculated explicitly based upon the volume of service water heating required and the entering makeup water and the leaving service water-heating temperatures. Entering water temperatures shall be estimated based upon the location. Leaving temperatures shall be based upon the end-use  requirements."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE  
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (e)  
+
+**Evaluation Context:** B_RMD - each building  
+**Data Lookup:**   
+**Function Call:**  
+- NONE  
+
+**Applicability Checks:**  
+- check that there is SWH in the proposed model AND that the building has a 24hr operating schedule
```

Comment:
I think this needs to be updated. The rule is not dependent on 24-hour operation.